### PR TITLE
B aws endpoint

### PR DESCRIPTION
**Pull request by vigilantee**
In reference to the AWS sign in flow for multi server authentication.

### CODE REVIEW

**Review by hrwX — APPROVED** [review]

**Comment by hrwX** [comment]
@vigilantee Fix the name of the PR as well as the commit messages for this to be merged.

File: frappe/utils/oauth.py [hunk 1]
```diff
@@ -232,7 +232,7 @@ def update_oauth_user(user, data, provider):
 		user.update({
 			"doctype":"User",
 			"first_name": get_first_name(data) or get_email(data),
-			"last_name": get_last_name(data),
+			"last_name": get_last_name(data) or "",
```
Comment (hrwX) [inline frappe/utils/oauth.py:235]:
```suggestion
			"last_name": get_last_name(data),
```
Return empty string from the get_last_name function

**Review by hrwX — CHANGES_REQUESTED** [review]

**Comment by neilLasrado** [comment]
@vigilantee Can you please implement the changes suggested by Himanshu? and fix your commit messages

**Comment by neilLasrado** [comment]
@vigilantee whats going on with this PR?

**Comment by hrwX** [comment]
@vigilantee Please make the changes asap, else I will be closing this PR

**Comment by neilLasrado** [comment]
@vigilantee what is going on with this?

**Comment by vigilantee** [comment]
@neilLasrado @hrwX closing this PR as the merge won't be required as last name is not mandatory for aws auth flow now.

File: frappe/utils/oauth.py (hunk 1 above)
Comment (vigilantee) [reply frappe/utils/oauth.py:235]: @hrwX Not required in the current aws login flow. So we can close this PR.

**Pull request closed without merging**